Clickable Images, Update Training events, New twist animation with alpha transparency support

milljm opened the pull request:
Add clickable images.
Use new transparent twist animations.
Add clickable 2025 conference banner.

Closes #26702

Commit:
Clickable Images, Transparent twist, conference

Add clickable images.
Use new transparent twist animations.
Add clickable 2025 conference banner.

DO NOT MERGE:
The link to the conference has yet to be decided.

Closes #26702

cticenhour commented on python/MooseDocs/extensions/media.py:
```diff
@@ -172,6 +173,7 @@ def defaultSettings():
         settings['tstart'] = (None, "Time (sec) to start video.")
         settings['tstop'] = (None, "Time (sec) to stop video.")
         settings['poster'] = (None, "Add a 'poster' image the the video")
+        settings['quicktime'] = (None, "Video to utilize Macintosh codecs (for alpha transparencies)")
```
Using the `video/quicktime` tag type doesn't cause an issue on non-Mac systems?

milljm replied:
nope nope... this is "the way" according to the masses. Ref: https://www.w3schools.com/html/html5_video.asp

Basically the web browser will continue down the list till it find a codec it likes...

cticenhour replied:
Nifty!

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 47379e241204355cd5701027bf85f222718162ec
+Subproject commit ca2f01ec0d781e845fa8171289b367e2497e6404
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2042320/) on b96e1c7 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/26716/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2042364/) on b96e1c7 wanted to post the following:


## Framework coverage

Coverage did not change


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/26716/coverage/xfem)

</details>



This comment will be updated on new commits.

Commit:
Update training page with new events

Adding MOOSE and PHYSOR training events.

Removing 2025 conference information as the landing page will likely not
be ready for some time (from what I have been told). But I would like to
get the 'clickable images' code in, as well as the new Safari compatible
twist animation with alpha channel.

Closes #26756 #26702

cticenhour commented on python/MooseDocs/extensions/media.py:
```diff
@@ -99,6 +99,7 @@ def defaultSettings():
         settings = command.CommandComponent.defaultSettings()
         settings['latex_src'] = (None, "Image to utilize when rendering with LaTeX")
         settings['dark_src'] = (None, "Image to utilize with dark HTML theme")
+        settings['link'] = (None, "Anchor to navigate to upon being clicked")
```
```suggestion
        settings['link'] = (None, "Anchor URL to navigate to upon being clicked")
```

Does this make more sense? At a glance, one might not know what "anchor" means.

cticenhour commented on python/MooseDocs/extensions/media.py:
```diff
@@ -205,15 +208,21 @@ def createHTML(self, parent, token, page):
 
-        pic = html.Tag(parent, 'picture')
+        if token['href']:
+            # Remove any styles being set for the img tag so it does not pollute the anchore tag
```
```suggestion
            # Remove any styles being set for the img tag so it does not pollute the anchor tag
```

cticenhour commented on modules/doc/content/training/more_detail/PHYSOR_MS_2024_04_21.md:
```diff
@@ -0,0 +1,32 @@
+The goal of the training session is to gain familiarity modeling the running-in phase of pebble bed
+reactors (PBRs) with the MOOSE-based tools Griffin and Pronghorn. Griffin includes capabilities for
+neutronics and depletion calculations and provides power densities to Pronghorn. Pronghorn uses the
+power densities to calculate temperatures using a porous medium flow model and user input
```
```suggestion
power densities to calculate temperatures using a porous medium flow model and user-input
```

cticenhour commented on modules/doc/content/training/more_detail/PHYSOR_PB_2024_04_21.md:
```diff
@@ -0,0 +1,32 @@
+The goal of the training session is to gain familiarity modeling the running-in phase of pebble bed
+reactors (PBRs) with the MOOSE-based tools Griffin and Pronghorn. Griffin includes capabilities for
+neutronics and depletion calculations and provides power densities to Pronghorn. Pronghorn uses the
+power densities to calculate temperatures using a porous medium flow model and user input
```
```suggestion
power densities to calculate temperatures using a porous medium flow model and user-input
```

cticenhour commented on modules/doc/content/training/more_detail/PHYSOR_MS_2024_04_21.md:
```diff
@@ -0,0 +1,32 @@
+
+!style! halign=left
+The goal of the training session is to gain familiarity modeling the running-in phase of pebble bed
+reactors (PBRs) with the MOOSE-based tools Griffin and Pronghorn. Griffin includes capabilities for
```
I think this may be using the wrong description? Both mention PBRs, while the title of this page refers to MSRs.

milljm replied:
Tell me about it... these descriptions are direct copies from: https://ncrcaims.inl.gov/Identity/Account/TrainingRegistration

I'll make the change regardless. 🍻

milljm replied:
Actually you are right, good catch! I copied the same text by mistake or something...

Commit:
Correct PBR description, comments, spelling

Address duplicate description for MSR and PBR training descriptions.
Correct spelling in hints, comments.

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit fd4e77d5249ab0cf5614ef03ec589f16a199124d
+Subproject commit 199a23c71d556343cbac89ceffbc987428aa4464
```
**Caution!** This contains a submodule update
